### PR TITLE
Adding monitoring guide starter

### PR DESCRIPTION
**Pull request by etsauer**
#### What is this PR About?
Adds a monitoring guide with pointers to things to health check a cluster.

#### How should we test or review this PR?
Build site, read through 0.0.0.0:4000/playbooks/operationalization/monitoring_guide.html

#### Is there a relevant Trello card or Github issue open for this?
https://trello.com/c/QtSyTv0v/9-3-existing-monitoring

#### Who would you like to review this?
cc: @redhat-cop/cant-contain-this

### CODE REVIEW

**Comment by sabre1041** [comment]
@etsauer This looks good and provides a nice baseline for future contribution. Only suggestion I would make is to change the raw links to the support scripts to a Hyperlink in the API endpoints section. (Such as [Script](https://github.com/redhat-cop/openshift-toolkit/blob/master/health_check/metrics-health-check.sh))

File: playbooks/operationalizing/monitoring_guide.adoc [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+Docker is an essential component of an OpenShift environment. The overall health of docker on each master and node instance ensures stability within an OpenShift cluster. The following components are areas that should be monitored on each node.
+
+.Docker checks
+[width="100%",cols="3,6,10",options="header"]
+|=========================================================
+|Check Name |Description |Sample Alerting Logic
+
+|Docker Daemon |Check that docker is running on a system | `systemctl is-active docker`
+
+|Docker Storage|Check that docker's local volume is not full | `echo "$(lvs \| awk '/docker-pool/ {print $5}') > 70" \| bc`
```
Comment (oybed) [inline playbooks/operationalizing/monitoring_guide.adoc]: Although this works, I'd also recommend looking at the output from the `docker info` command - i.e.: as done here for Nagios:
https://github.com/redhat-cop/casl-ansible/blob/master/roles/nagios-target/files/plugins/check_docker_storage

File: playbooks/operationalizing/monitoring_guide.adoc [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+== Ensuring a cluster has adequate capacity
+
+.Feedback or Contribution Needed
+****
+Need a cluster capacity monitoring guide.
+****
+
+== What's Next?
+
+* Monitoring Tooling Implementations (Coming Soon!)
```
Comment (oybed) [inline playbooks/operationalizing/monitoring_guide.adoc]: A partial implementation for Nagios can be found here:
https://github.com/redhat-cop/casl-ansible/tree/master/roles/nagios-target
(includes more than just OpenShift)

**Comment by etsauer** [comment]
@oybed @sabre1041 updated based on your feedback.

**Comment by sabre1041** [comment]
@etsauer lgtm

Will wait to hear back from @oybed on his review

**Pull request merged**